src/Components/Workout/Workout.js

```diff
@@ -1,12 +1,52 @@
 import React from "react";
 
-import "./Workout.css";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import Card from "@material-ui/core/Card";
+import Grid from "@material-ui/core/Grid";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardMedia from "@material-ui/core/CardMedia";
+
+const useStyles = makeStyles((theme) => ({
+  workoutCard: {
+    margin: "1em",
+    [theme.breakpoints.down("sm")]: {
+      marginBottom: "1em",
+    },
+  },
+  root: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 200,
+  },
+  cardText: {
+    textAlign: "center",
+  },
+}));
+
+export const Workout = ({ name, description, image }) => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const matchesSM = useMediaQuery(theme.breakpoints.down("sm"));
```

Comment:
I noticed we are not using the matchesSM variable yet, do we need it right now or...?

Reply:
I initially had the components all in one .js file. I think I forgot to remove that from the Workout file. I will also do this next. :)